spec/support/logger_helper.rb

```diff
@@ -0,0 +1,9 @@
+module LoggerHelper
+  def logger
```

Comment:
make the rails logger easily available in all specs

Reply:
Is this worth the extra effort/indirection? 😅  I don't see the logger helper used anywhere yet, I think? 🤔